libs/frontend/ui-widgets/src/lib/form/PSMultiselect.vue

```diff
@@ -97,6 +100,10 @@ const props = withDefaults(
     tooltip?: string
     emptyText?: string
     openInitial?: boolean
+    // If true, the label is replaced with a slot
+    customLabel?: boolean
+    // Allow any search input to be selected, even if it's not in the list
+    allowAny?: boolean
```

Comment:
@abemscac I added this so it's possible to set a Google font which is not included in the list. It seems to work, but let me know if you run into any problems with fonts!